test/jest/roktManager.spec.ts

```diff
@@ -17,17 +19,85 @@ describe('RoktManager', () => {
         });
     });
 
+    describe('#init', () => {
+        it('should initialize the manager with defaults when no config is provided', () => {
+            roktManager.init({} as SDKInitConfig);
+            expect(roktManager['launcher']).toBeNull();
+            expect(roktManager['filters']).toEqual({
+                userAttributeFilters: undefined,
+                filterUserAttributes: expect.any(Function),
+            });
+            expect(roktManager['filteredUser']).toBeUndefined();
+            expect(roktManager['kit']).toBeNull();
+        });
+        it('should initialize the manager with user attribute filters from a config', () => {
+            const kitConfig: Partial<IKitConfigs> = {
+                name: 'Rokt',
+                moduleId: 181,
+                userAttributeFilters: [816506310, 1463937872, 36300687],
+            };
+
+            const config: SDKInitConfig = {
+                kitConfigs: [kitConfig as IKitConfigs],
+            };
+
+            roktManager.init(config);
+            expect(roktManager['filters']).toEqual({
+                userAttributeFilters: [816506310, 1463937872, 36300687],
+                filterUserAttributes: expect.any(Function),
+            });
+        });
+    });
+
+    describe('#parseConfig', () => {
+        it('should ONLY return the kit config for Rokt', () => {
+            const kitConfig: Partial<IKitConfigs> = {
+                name: 'Rokt',
+                moduleId: 181,
+            };
+
+            const otherKitConfig: Partial<IKitConfigs> = {
+                name: 'Other Kit',
+                moduleId: 42,
+            };
+
+            const config: SDKInitConfig = {
+                kitConfigs: [kitConfig as IKitConfigs, otherKitConfig as IKitConfigs],
+            };
+
+            const result = roktManager.parseConfig(config);
+            expect(result).toEqual(kitConfig);
+        });
+
+        it('should return undefined if no kit config is found', () => {
+            const config: SDKInitConfig = {
+                kitConfigs: [],
+            };
+
+            const result = roktManager.parseConfig(config);
+            expect(result).toBeUndefined();
+        });
+    });
+
     describe('#attachLauncher', () => {
```

Comment:
as above, I don't think we need to attach the launcher to the core anymore, so this entire test block should be removed when the code is removed in the core sdk above